packages/wrangler/src/publish.ts

```diff
@@ -61,7 +60,10 @@ export default async function publish(props: Props): Promise<void> {
   }
 
   let scriptName = props.script ? props.name : config.name;
-  scriptName += props.env ? `-${props.env}` : "";
+  if (props.legacyEnv) {
+    scriptName += props.env ? `-${props.env}` : "";
+  }
+  const envName = props.env ?? "production";
```

Comment:
What if `production` is not the default or if they renamed it?

Reply:
^ good question! what's the plan here.

Reply:
Good point, I think we'll need to support some type of `/services/:name/environments/~default`

Reply:
We can just use `/services/:name` for default, right?